### PR TITLE
APIC quickstart

### PR DESCRIPTION
**Pull request by jesusmah**

### CODE REVIEW

**Comment by hollisc** [comment]
@jesusmah, looks good.  You should also add `ibm-apic-instance.yaml` to the `other` profiles as well and the update to kustomization.yaml for `2-services`

File: 0-bootstrap/single-cluster/2-services/argocd/instances/ibm-apic-instance.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        ibm-apic-instance:
+          ibmapicinstance:
+            name: apic-cluster
+            spec:
+              license:
+                accept: true
+                use: production
+                license: L-RJON-BZEP9N
+              version: 10.0.3.0-ifix1
+              profile: n3xc14.m48
```
Comment (csantanapr) [inline 0-bootstrap/single-cluster/2-services/argocd/instances/ibm-apic-instance.yaml:34]: Is this a ROKS specific machine size?

Comment (csantanapr) [reply 0-bootstrap/single-cluster/2-services/argocd/instances/ibm-apic-instance.yaml:34]: I think I found it in the docs.
Maybe we can put in the recipe docs
```
n1xc10.m48
Deploys 1 replica of each pod, so this profile is most suitable for a small, non-HA system. Recommended use of this profile is for development and testing.

n3xc14.m48
Deploys 3 or more replicas of each pod, so this profile is most suitable for larger systems and for production environments. This profile is supported for installation on a cluster with three or more nodes. It is not supported on a cluster with fewer than three nodes.
```

**Review by hollisc — APPROVED** [review]
Looks good

**Pull request merged**